### PR TITLE
 [15.0][IMP]  account-reconcile: test performance improvement 

### PR DESCRIPTION
**Pull request by josep-tecnativa**
CC @Tecnativa TT45458

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please check CI

File: account_reconciliation_widget_due_date/tests/test_account_reconciliation_widget_date_due.py [hunk 1]
```diff
@@ -7,29 +7,40 @@
...
-                "property_account_receivable_id": self.account.id,
+                "property_account_receivable_id": cls.account.id,
             }
         )
-        self.partner_b = self.partner_a.copy({"name": "Partner Test B"})
-        self.partner_c = self.partner_a.copy({"name": "Partner Test C"})
-        self.statement = self._create_account_bank_statement()
+        cls.partner_b = cls.partner_a.copy({"name": "Partner Test B"})
+        cls.partner_c = cls.partner_a.copy({"name": "Partner Test C"})
+        cls.statement = cls._create_account_bank_statement(cls)
```
Comment (pedrobaeza) [inline account_reconciliation_widget_due_date/tests/test_account_reconciliation_widget_date_due.py]: A better option here is to convert `_create_account_bank_statement` to `@classmethod`.

**Comment by josep-tecnativa** [comment]
ping @pedrobaeza

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge patch

**Pull request merged**